### DISCUSSION TITLE
More binary distributions?

### DISCUSSION THREAD

**Discussion by baentsch** [open-quantum-safe/liboqs#936] [Ideas]
As per [this](https://github.com/open-quantum-safe/liboqs/issues/935#issuecomment-792338085) do we want to consider making binary distros available? For which platforms? For which languages/wrappers?

**Comment by xvzcf** [comment]
I think I'm against this in general, but we can discuss more in the status meeting.

**Reply by baentsch** [reply]
I guess this also was the general consensus. Is there a way to close discussions?
